test/cli/workspace/create/expected-stdout

```diff
@@ -0,0 +1 @@
+Created new workspace "foo".
```

Comment:
Mozno by tu mohla byt este hlaska `Creating workspace, please wait.`, kedze uz vieme, ze to moze byt `5min+`.

Reply:
Přidáno